### ISSUE TITLE
Need a cleaner separation between `preview` and `release` builds

### ISSUE THREAD

**Issue by floatplane** [NWACus/avy#563]
I downloaded a build from the app store and launched and it immediately crashed, which freaked me out! but:
- the app store is still serving up the old `release` version, 1.0.1
- TestFlight had already installed the pending `release` version, 1.0.2
and I think Expo was trying to run the new JS on top of the old build. A clean uninstall and reinstall worked fine.

This illustrates that `preview` and `release` builds are getting interleaved into the same TestFlight channel, and that’s sure to keep causing headaches. We should make `preview` kick out an app with a different name and bundle id ("Avy test" or something) just to keep things sane. Give it a different icon with a beta badge for good measure. 

In other words, you should be able to use TestFlight to deploy `preview` builds ("here's what's coming in the next few weeks") and `release` builds ("here's the thing we're going to deploy today") and not have them interfere with each other.

**Comment by rustynwac** [comment]
Charlotte thinks this might be ready to close...?

**Comment by yuliadub** [comment]
@rustynwac yup, sounds right - when we switched to using two apps, one for production and one for testing, we resolved the concern Brian was listing here with the builds and expo channels since we used to just use one app for both production and testing. @stevekuznetsov keep me honest though

**Comment by stevekuznetsov** [comment]
Yep, the preview app solved this.
